test/index.test.js

```diff
@@ -93,4 +94,78 @@ describe("@probot/adapter-aws-lambda-serverless", () => {
 
     expect(mock.activeMocks()).toStrictEqual([]);
   });
+
+  test("lowercase request headers", async () => {
+    const fn = createLambdaFunction(app, { probot });
+
+    const mock = nock("https://api.github.com")
+      .post(
+        "/repos/probot/adapter-adapter-aws-lambda-serverless/commits/headcommitsha123/comments",
+        (requestBody) => {
+          expect(requestBody).toStrictEqual({
+            body: `Hello from test${path.sep}fixtures${path.sep}app.js`,
+          });
+
+          return true;
+        }
+      )
+      .reply(201, {});
+
+    const context = {};
+    const payload = JSON.stringify(require("./fixtures/push.json"));
+    const signature = probot.webhooks.sign(payload);
+    const event = {
+      headers: {
+        "x-github-delivery": "eventid123",
+        "x-github-event": "push",
+        "x-hub-signature": signature,
+      },
+      body: payload,
+    };
+
+    await fn(event, context);
+
+    expect(context).toStrictEqual({
+      callbackWaitsForEmptyEventLoop: false,
+    });
+
+    expect(mock.activeMocks()).toStrictEqual([]);
+  });
+
+  test("camelcase request headers", async () => {
```

Comment:
Maybe add a reference to the issue as it includes relevant background information about the casing of the headers in this test

```suggestion
  test("camelcase request headers (#62)", async () => {
```